product_warranty/i18n/es_MX.po

```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* product_warranty
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
```

Comment:
To avoid the warning "es_**" do not have translations, on this case there is not any different terms, but without the file the warning appears always. We use it es_MX and es_PA